src/BaseBuilder.js

```diff
@@ -4,6 +4,11 @@ import tcomb from 'tcomb-validation';
 import { validation } from './combinators';
 import * as validators from './validators';
 
+function keyIn(...keys) {
```

Comment:
Can you put this in the utils file? https://github.com/cadre/tcomb-builder/blob/master/src/utils.js